### PR TITLE
Minor documentation updates.

### PR DESCRIPTION
**Pull request by bradh**

### CODE REVIEW

**Comment by dionhaefner** [comment]
This looks great, thanks!

I’m currently on vacation so it might take a while for me to review this, just a heads-up (but maybe someone else will have the capacity).

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Optimizing rasters: 100%|██████████████████
 ### 2. Create a database from file name pattern
 
 ```bash
-$ terracotta ingest optmized/S2A_{date}_{}_{tile}_{band}.tif -o greenland.sqlite
+$ terracotta ingest optimized/S2A_{date}_{}_{tile}_{band}.tif -o greenland.sqlite
```
Comment (bradh) [inline README.md]: @mrpgraae Thanks.

**Comment by bradh** [comment]
@mrpgraae Thanks for the fixes - clearly not my finest docs work, but glad to be able to offer a little.

**Comment by mrpgraae** [comment]
> 
> 
> @mrpgraae Thanks for the fixes - clearly not my finest docs work, but glad to be able to offer a little.

@bradh just to clarify, when I say "Some needed changes and additions for sure" I meant that _you_ provided some needed changes and additions to our docs 😄

**Pull request merged**